examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -210,9 +210,10 @@ extern "C" void app_main()
     {
         ESP_LOGE(TAG, "GetAppTask().StartAppTask() failed : %" CHIP_ERROR_FORMAT, error.Format());
     }
-    ESPOpenThreadInit();
 
     chip::DeviceLayer::PlatformMgr().ScheduleWork(InitServer, reinterpret_cast<intptr_t>(nullptr));
+
+    ESPOpenThreadInit();
```

Comment:
I don't get this change.  InitServer runs async, so nothing guarantees it will finish, or even start, before ESPOpenThreadInit is called.  @Jerry-ESP what am I missing?

Reply:
Have raised a new PR to fix this: https://github.com/project-chip/connectedhomeip/pull/37073